doc/sphinx/user/cookbooks/geophysical-setups.md

```diff
@@ -106,5 +106,6 @@ cookbooks/global_melt/doc/global_melt.md
 cookbooks/mid_ocean_ridge/doc/mid_ocean_ridge.md
 cookbooks/kinematically_driven_subduction_2d/doc/kinematically_driven_subduction_2d.md
 cookbooks/inclusions/doc/inclusions.md
+cookbooks/equilibrium_grain_size/doc/tomography_based_model.md
```

Comment:
Here you name the section 'tomography based model', but the folder is called 'equilibrium grain size'. I think both are not very good names for this cookbook (tomography based model is quite unspecific, and the model has not much to do with grain size anymore). I would suggest: "Tomography based plate motions" or "tomography_based_plate_motions.prm. I know this is a lot of work, but please rename the folder and the file, and all references to the old names in all files (search and replace is your friend :-)).

Reply:
Got it, thanks for suggestion!